React InstantSearch Hooks Native/getting-started/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "ais-ecommerce-demo-app",
+  "version": "1.0.0",
+  "private": true,
+  "main": "node_modules/expo/AppEntry.js",
+  "scripts": {
+    "start": "expo start",
+    "android": "expo start --android",
+    "ios": "expo start --ios",
+    "web": "expo start --web",
+    "eject": "expo eject"
+  },
+  "dependencies": {
+    "algoliasearch": "4.12.1",
+    "expo": "~44.0.0",
+    "expo-status-bar": "~1.2.0",
+    "instantsearch.js": "4.38.1",
+    "react": "17.0.1",
+    "react-dom": "17.0.1",
+    "react-instantsearch-hooks": "6.22.0",
+    "react-native": "0.64.3",
+    "react-native-web": "0.17.1"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.12.9",
+    "@types/react": "~17.0.21",
+    "@types/react-native": "~0.64.12",
+    "typescript": "~4.3.5"
```

Comment:
We could add `expo-cli` here, instead of having to install it globally by ourselves.

Reply:
Yes it's better, addressed in [f3e8087](https://github.com/algolia/doc-code-samples/pull/358/commits/f3e8087f9a2fffccff8515a80cb75ad04e5e7f8e).